.github/workflows/test.yml

```diff
@@ -30,8 +30,10 @@ jobs:
         run: forge test -vvv
         env:
           RPC_MAINNET: ${{ secrets.RPC_MAINNET }}
+          CHAIN_ID: ${{ secrets.CHAIN_ID }}
```

Comment:
seems odd we're making the chain ID a secret?
I feel perhaps OK doing this for now but would like to see us get a better solution for this...

Reply:
I think its ok for now since we don't want to run fork tests in CI.